journal/src/main/scala/com/github/j5ik2o/akka/persistence/dynamodb/serialization/ByteArrayJournalSerializer.scala

```diff
@@ -89,37 +91,38 @@ class ByteArrayJournalSerializer(
       System.currentTimeMillis(),
       encodeTags(tags, separator)
     )
+    val traced = traceReporter.fold(future)(_.traceJournalSerializeJournal(context)(future))
 
-    future.onComplete {
+    traced.onComplete {
       case Success(_) =>
         metricsReporter.foreach(_.afterJournalSerializeJournal(newContext))
       case Failure(ex) =>
         metricsReporter.foreach(_.errorJournalSerializeJournal(newContext, ex))
     }
-
-    future
+    traced
   }
 
   override def deserialize(
       journalRow: JournalRow
   )(implicit ec: ExecutionContext): Future[(PersistentRepr, Set[String], Long)] = {
     val pid        = journalRow.persistenceId
-    val context    = MetricsReporter.newContext(UUID.randomUUID(), pid)
+    val context    = Context.newContext(UUID.randomUUID(), pid)
     val newContext = metricsReporter.fold(context)(_.beforeJournalDeserializeJournal(context))
 
     val future = for {
```

Comment:
Change to def.

Reply:
fixed. 
https://github.com/j5ik2o/akka-persistence-dynamodb/pull/943/commits/91e2814a4fcf841b337569bf886e735fd5f6d93e#diff-599e2fb130750a3c53ab98d064e4eddec7688c71eb9548e4074946a727aace74R112